meetings/2020-11-16.md

```diff
@@ -1,13 +1,58 @@
 # ASWF Python 3 Working Group Meeting - 2020/11/16
 ## Attendees
+* Ryan Bottriell
+* David Aguilar
+* Ashley Whetter
+* Ollie HArding
+* Mark Wiebe
+* Rick Rickenberg
+* JT Nelson
+* Robert Vinluan
 
-# Action Items
+## Action Items
```

Comment:
Please can you also add an action item for me to add a note to the best practices about the minimum version of modernize that needs to be used to fix the `long -> int` conversion in `maya.cmds` function calls once a section about modernize has been added to the guide.